### PR TITLE
KIALI-2755 Add more quoting in the makefiles

### PR DESCRIPTION
**Pull request by jshaughn**
Changes such that Makefiles work better in certain bash envs

### CODE REVIEW

**Comment by rhqci** [comment]
Jenkins CI: kiali-core-pr-e2e-test [#967](https://paste.fedoraproject.org/paste/JsKPi4Vla1xFTE1lCr51ag)
  * :heavy_check_mark: run-kiali-e2e-tests [#[1778]](https://paste.fedoraproject.org/paste/27UvfvnjrkrY7vIZluwj~w)

File: operator/Makefile [hunk 1]
```diff
@@ -38,7 +38,7 @@ help: Makefile
 # Requires operator-sdk - download it from https://github.com/operator-framework/operator-sdk/releases
 operator-build: .ensure-operator-sdk-exists
 	@echo Build operator
-	${OP_SDK} build ${OPERATOR_IMAGE_NAME}:${OPERATOR_IMAGE_VERSION}
+	${OP_SDK} build "${OPERATOR_IMAGE_NAME}":${OPERATOR_IMAGE_VERSION}
```
Comment (jmazzitelli) [inline operator/Makefile]: why wouldn't you wrap the whole thing in quotes?
```suggestion
	${OP_SDK} build "${OPERATOR_IMAGE_NAME}:${OPERATOR_IMAGE_VERSION}"
```

File: operator/Makefile [hunk 2]
```diff
@@ -77,19 +77,19 @@ deploy/deploy-kiali-operator.sh
 ## operator-delete: Remove the Kiali operator resources from the cluster along with Kiali itself
 operator-delete: purge-kiali
 	@echo Remove Operator
-	${OC} delete --ignore-not-found=true all,sa,deployments,clusterroles,clusterrolebindings,customresourcedefinitions --selector=app=kiali-operator -n ${OPERATOR_NAMESPACE}
```
Comment (jmazzitelli) [inline operator/Makefile]: put back the "--ignore-not-found=true" option

File: operator/Makefile (hunk 1 above)
Comment (jshaughn) [reply operator/Makefile]: I did what I needed to do to get it working but enclosing the whole thing should be fine, I'll change it.

File: operator/Makefile (hunk 2 above)
Comment (jshaughn) [reply operator/Makefile]: yes.

**Comment by rhqci** [comment]
Jenkins CI: kiali-core-pr-e2e-test [#969](https://paste.fedoraproject.org/paste/b2nWlop7XuyCHXArBlNhSw)
  * :heavy_check_mark: run-kiali-e2e-tests [#[1780]](https://paste.fedoraproject.org/paste/Q4hbulvAWibt~lx3V65Prg)

File: operator/Makefile [hunk 3]
```diff
@@ -104,19 +104,19 @@ IMAGE_VERSION=${IMAGE_VERSION} \
...
 
 ## purge-kiali: Purges all Kiali resources directly without going through the operator or ansible.
 purge-kiali:
 	@echo Purge Kiali resources
-	${OC} patch kiali kiali -n ${OPERATOR_NAMESPACE} -p '{"metadata":{"finalizers": []}}' --type=merge ; true
-	${OC} delete --ignore-not-found=true all,secrets,sa,templates,configmaps,deployments,clusterroles,clusterrolebindings,ingresses,customresourcedefinitions --selector=app=kiali -n ${NAMESPACE}
-	${OC} delete --ignore-not-found=true oauthclients.oauth.openshift.io --selector=app=kiali -n ${NAMESPACE} ; true
+	${OC} patch kiali kiali -n "${OPERATOR_NAMESPACE}" -p '{"metadata":{"finalizers": []}}' --type=merge ; true
+	${OC} delete --ignore-not-found=true all,secrets,sa,templates,configmaps,deployments,clusterroles,clusterrolebindings,ingresses,customresourcedefinitions --selector=app=kiali -n "${NAMESPACE}"
+	${OC} delete --ignore-not-found=true oauthclients.oauth.openshift.io --selector=app=kiali -n "${NAMESPACE}" ; true
```
Comment (jmazzitelli) [inline operator/Makefile]: why isn't selector option in these two commands also quoted? Seems like you needed to quote them elsewhere, but they are not here.. would it be best/consistent to say --selector="app=kiali" in these two lines too?

File: operator/Makefile [hunk 4]
```diff
@@ -38,7 +38,7 @@ help: Makefile
 # Requires operator-sdk - download it from https://github.com/operator-framework/operator-sdk/releases
 operator-build: .ensure-operator-sdk-exists
 	@echo Build operator
-	${OP_SDK} build ${OPERATOR_IMAGE_NAME}:${OPERATOR_IMAGE_VERSION}
+	${OP_SDK} build "${OPERATOR_IMAGE_NAME}:${OPERATOR_IMAGE_VERSION}"
```
Comment (jmazzitelli) [inline operator/Makefile]: you quoted the image name/version here, but not a few lines down (line 46) in docker push command - should these be quoted consistently?

File: operator/Makefile (hunk 3 above)
Comment (jshaughn) [reply operator/Makefile]: Yes.

File: operator/Makefile (hunk 4 above)
Comment (jshaughn) [reply operator/Makefile]: I suppose.

**Comment by rhqci** [comment]
PRT Failed, Please Contact QE

**Comment by rhqci** [comment]
Jenkins CI: kiali-core-pr-e2e-test [#975](https://paste.fedoraproject.org/paste/e0Puzb4yG-rzWvt-VXd3Ng)
  * :heavy_check_mark: run-kiali-e2e-tests [#[1788]](https://paste.fedoraproject.org/paste/QZqQ4~V8UinH6kP2ALOQHw)

**Pull request merged**